src/templates/template.cpp

```diff
@@ -604,6 +604,18 @@ void Template::unloadTemplateFile()
 	emit templateStateChanged();
 }
 
+void Template::releaseGeoreference()
+{
+	// Calling this method on an unloaded template results in
+	// the template position being set to (0, 0, 1, 1, 0).
+	// XXX This assert is insufficient. FIXME!!!
```

Comment:
The assert is okay for me ATM if the precondition is stated in the documentation. (We have a number of methods like this.)

Reply:
I could not get the assert to trigger, hence the FIXME comment. In other words, I could still release an unloaded template.

Reply:
Originally, I was looking for a way to load an unloaded template here. I think that is a better option. Is there an easy way to accomplish this?

Reply:
It actually just a call to `loadTemplateFile(false)`.